pkg/transaction/builder_test.go

```diff
@@ -76,3 +76,98 @@ func TestMakeSTXTokenTransfer(t *testing.T) {
 		t.Errorf("Expected nonce %d, got %d", specifiedNonce.Uint64(), tx2.Auth.OriginAuth.Nonce)
 	}
 }
+
```

Comment:
Any reason this is commented?

Reply:
The broadcast tests written don't really belong there because they aren't mocked. So, they are integration tests, not unit tests, and I didn't want to submit tx to Stacks every time `go test` is run.

Reply:
I moved them to an integration test folder and uncommented them. https://github.com/icon-project/stacks-go-sdk/pull/17/commits/0ab00ee1b6adfd0edf542db67e13fbcbffe87304